src/models/user.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, Integer, String, DateTime
+from sqlalchemy.orm import relationship
+
+from src.db.db import Base
+
+
+class User(Base):
+    # classes will inherit from Base
+    __tablename__ = "users"
+    # set tablename for use in db
```

Comment:
You can delete these comments.

Reply:
They're more for my notes but I can delete.